contributors/contributors/social-engagement-guidelines.md

```diff
@@ -0,0 +1,150 @@
+
+6. **Do your best to link documentation as answers**
+
+* Links to documentation allow answers to be easily updated dynamically as documentation is updated.
+* Any questions that should be answered in docs that aren’t should turn into [tickets to create that documentation](https://github.com/mattermost/docs/issues) (with a link to the ticket included in the response).
+
+7. **Keep community end-user information secure**
+
+* If you come across a post that includes the person's IP address, domain name, or other information you think should not be disclosed publicly, edit the post to remove this information. Then click the hide revision button so that your edits won't be visible to others on the forum.
+
+8. **Be welcoming and thankful**
+
+* When asking someone to take action, use "Would you be open to..." instead of "Would you like to...".
+* Communities really respond well to being praised and thanked for their work.
+
+## Community Forums
+
+Below are guidelines or processes for specific forums we engage with the Mattermost community:
+
+### Twitter
+
+**Intended Outcomes**
+
+1. **Likes and loves:** Every promoter feels appreciated (like, heart, etc.) and we have acknowledgement to the best content (re-tweet, quoted re-tweet, mention, shout out, replies, * humor, etc.)
+2. **Listening and respond:** Everyone with a relevant question gets a reply in 1 business day (except trolls)
+3. **Data, learning, and iteration:** Understand how we're doing week-to-week, what we've learned, and how we'll up-level our approach.
+
+**Workflow**
+
+1. Monitor [Twitter channel in Mattermost Contributors Team](https://community.mattermost.com/core/channels/twitter) and [this search query](https://twitter.com/search?q=mattermost&src=typed_query&f=live).
+2. Respond to questions, retweet, and like posts that we are tagged in per the guidelines outlined above.
+* For product inquiries you are uncertain how to answer, you may escalate by at-mentioning `@pmteam` in the Twitter channel header.
+3. Update posts in the Twitter channel with action taken.
+* React with `:heart:` if you liked the post.
+* React with `:repeat:` if you retweeted the post.
+* React with `:no_entry_sign:` if you took no action.
+
+**Response writing tips**
+
+1. **Avoid acronyms.** Avoid acronyms when possible. For example, say "pull request" instead of "PR", since only a subset of tweet readers are active GitHub users.
+2. **Use the active voice.** Avoid "has", "was", "have been" when possible. For example, instead of "Hackfest has started!" say "Hackfest starts now!"
+3. **Include at most one link.** To provide a clear call to action, include at most one link per tweet and place it near the end of the tweet.
+4. **Use exclamation marks only for exciting announcements.** An exclamation mark can be used when the announcement is exciting, but using an exclamation mark should be avoided when it can be confused with a signal for cthe ommunity to panic, e.g. "Security update released!".
```

Comment:
```suggestion
4. **Use exclamation marks only for exciting announcements.** An exclamation mark can be used when the announcement is exciting, but using an exclamation mark should be avoided when it can be confused with a signal for the community to panic, e.g. "Security update released!".
```